app/src/main/java/org/fossasia/openevent/api/processor/MicrolocationListResponseProcessor.java

```diff
@@ -13,12 +16,29 @@
  */
 public class MicrolocationListResponseProcessor implements Callback<MicrolocationResponseList> {
     private static final String TAG = "Microprocessor";
+    ArrayList<String> queries = new ArrayList<String>();
 
     @Override
     public void success(MicrolocationResponseList microlocationResponseList, Response response) {
+        for (Microlocation microlocation : microlocationResponseList.microlocations)
+
+        {
+            int id = microlocation.getId();
+            String name = microlocation.getName();
+            float latitude = microlocation.getLatitude();
+            float longitude = microlocation.getLongitude();
+            int floor = microlocation.getFloor();
+            Microlocation temp = new Microlocation(id, name, latitude, longitude, floor);
```

Comment:
I don't understand the reasoning? Writing more code => easy to understand? I dont get it. 
Also 
>Yes I can but I made exactly like I made others 

Can you tell me what is `others` in this context?